guide.html

```diff
@@ -0,0 +1,425 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+    <meta name="description" content="Dogecoin is an open source peer-to-peer digital currency, favored by Shiba Inus worldwide." />
+    <meta name="author" content="" />
+        <!--
+               __                            _
+          ____/ /___  ____ ____  _________  (_)___
+         / __  / __ \/ __ `/ _ \/ ___/ __ \/ / __ \
+        / /_/ / /_/ / /_/ /  __/ /__/ /_/ / / / / /
+        \__,_/\____/\__, /\___/\___/\____/_/_/ /_/
+                   /____/
+
+        so markup. very website. wow.
+        made with love by edswf1
+    -->
+    <title>Dogecoin| Getting started Guide</title>
+    <!-- Favicon-->
+    <link rel="icon" type="image/x-icon" href="assets/img/favicon.ico" />
+    <!-- preload webfonts -->
+    <link rel="preload" href="assets/vendor/fa/webfonts/fa-solid-900.woff2" as="font" type="font/woff2" crossorigin="anonymous">
+    <link rel="preload" href="assets/vendor/fa/webfonts/fa-brands-400.woff2" as="font" type="font/woff2" crossorigin="anonymous">
+    <link rel="stylesheet" href="assets/vendor/fa/css/all.min.css"/>
+    <!-- Google fonts-->
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700&display=swap" rel="stylesheet" type="text/css" />
+    <link href="https://fonts.googleapis.com/css?family=Lato:400,700,400italic,700italic&display=swap" rel="stylesheet" type="text/css" />
+    <!-- Core theme CSS (includes Bootstrap)-->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" rel="stylesheet" type="text/css" ></script>
+
+    <link href="assets/css/styles.css" rel="stylesheet" />
+</head>
+<body id="page-top">
+    <!-- Navigation-->
+    <nav class="navbar navbar-expand-lg bg-secondary text-uppercase fixed-top" id="mainNav">
+        <div class="container-fluid">
+            <a class="navbar-brand js-scroll-trigger" href="#page-top"><img src="assets/img/dogecoin-300.png">&nbsp;Dogecoin</a>
+            <button class="navbar-toggler navbar-toggler-right text-uppercase font-weight-bold bg-primary text-secondary rounded" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+                Menu
+                <i class="fas fa-bars"></i>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarResponsive">
+                <ul class="navbar-nav ml-auto">
+                    <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded" href="index.html" data-i18n="dgc-back-to-main">Back to Main Page</a></li>
+                    <li class="nav-item mx-0 mx-lg-1"><span class="nav-link py-3 px-0 px-lg-3 rounded text-white curPrice_USD"></span></li>
```

Comment:
this should be removed

Reply:
i think you are referring to the curPrice_USD element. removed that now. "Back to Main Page" is part of the menu and shouldn't be removed